dependency-inversion-android/README.md

```diff
@@ -0,0 +1,9 @@
+# Dependency Inversion
+
+Throughout this exercise YOU CANNOT CHANGE THE IMPLEMENTATIONS OF InternetFruitRepository or Preferences! However, you can make them implement interfaces.
+
+Go to the `CachingFruitFetcherTests`. First try to write the three tests without touching the current code. Hard, right?
+
+Now use dependency inversion on the _dependencies_ of the `fetchFruit()` method (there are two). Now can you write the tests?
```

Comment:
updated, including the slides I wrote for this :) https://github.com/novoda/dojos/pull/28/commits/841775d6e22cd9c82c4a0c548c7ace74014abfe2